src/pokemon.c

```diff
@@ -4123,7 +4123,9 @@ void CalculateMonStats(struct Pokemon *mon)
     s32 spDefenseIV = GetMonData(mon, MON_DATA_SPDEF_IV, NULL);
     s32 spDefenseEV = GetMonData(mon, MON_DATA_SPDEF_EV, NULL);
     u16 species = GetMonData(mon, MON_DATA_SPECIES, NULL);
+    #if B_FRIENDSHIP_BOOST == TRUE
```

Comment:
see previous comment about phrasing this as C `if` instead of preproc directive to cause the compiler to check all code paths.